lib/auth/providers.ts

```
@@ -0,0 +1,30 @@
+import { credentialsProvider } from "./credentials";
+import GoogleProvider from "next-auth/providers/google";
+import GitHubProvider from "next-auth/providers/github";
+import FacebookProvider from "next-auth/providers/facebook";
+import TwitterProvider from "next-auth/providers/twitter";
+import MicrosoftProvider from "next-auth/providers/azure-ad";
+
+export const providers = [
+  credentialsProvider,
+  GoogleProvider({
+    clientId: process.env.GOOGLE_CLIENT_ID!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)